journal/vendor-extra/ApiClient/ArticlesClient.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Libero\ApiClient;
+
+use GuzzleHttp\Promise\PromiseInterface;
+use GuzzleHttp\Psr7\Request;
+
+final class ArticlesClient
```

Comment:
Provides a client for the defined Libero articles API.

If you choose to extend the API spec, like https://github.com/libero/walking-skeleton/blob/e89141108b1f3a8208d852ebe5aeed801a730681/api-doc/api.raml#L20-L59 you'll need your own client. (_Might_ be something we could do about that?)